<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ZA-5CD350K5XK" failures="1" tests="11" name="tests.RegistrationPageTest" time="345.007" errors="0" timestamp="2024-08-19T15:05:36 SAST" skipped="0">
  <testcase classname="tests.RegistrationPageTest" name="verifyRegistration" time="32.863"/>
  <testcase classname="tests.RegistrationPageTest" name="verifyRegistration" time="31.409"/>
  <testcase classname="tests.RegistrationPageTest" name="verifyRegistration" time="53.773">
    <failure message="OTP not sent. expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: OTP not sent. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at tests.RegistrationPageTest.verifyRegistration(RegistrationPageTest.java:97)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- verifyRegistration -->
  <testcase classname="tests.RegistrationPageTest" name="verifyRegistration" time="31.326"/>
  <testcase classname="tests.RegistrationPageTest" name="verifyRegistration" time="27.655"/>
  <testcase classname="tests.RegistrationPageTest" name="verifyRegistration" time="27.909"/>
  <testcase classname="tests.RegistrationPageTest" name="verifyRegistration" time="27.923"/>
  <testcase classname="tests.RegistrationPageTest" name="verifyRegistration" time="28.322"/>
  <testcase classname="tests.RegistrationPageTest" name="verifyRegistration" time="27.708"/>
  <testcase classname="tests.RegistrationPageTest" name="verifyRegistration" time="28.081"/>
  <testcase classname="tests.RegistrationPageTest" name="verifyRegistration" time="28.038"/>
</testsuite> <!-- tests.RegistrationPageTest -->
